tests/func/experiments/test_remote.py

```diff
@@ -343,7 +343,7 @@ def test_auth_error_list(tmp_dir, scm, dvc, http_auth_patch):
 
     with pytest.raises(
         GitAuthError,
-        match=f"HTTP Git authentication is not supported: '{http_auth_patch}'",
+        match=f"Authentication failed for: '{http_auth_patch}'",
```

Comment:
Unrelated changes?

Reply:
This is related to scmrepo's new version. related to https://github.com/iterative/scmrepo/pull/159